Alexander96779/AFree-Mentors
# #168383507 Adding more pages to the UI

#### What does this PR do?

- This PR adds more pages to the UI

#### Description of Task to be completed?

- Adding a profile page and changing the session page

#### How should this be manually tested?

- Visit my gh pages and go to the signup page

- Fill in the info and then sign up

- A complete profile page is displayed

#### What are the relevant pivotal tracker stories?

-  [#168383507](https://www.pivotaltracker.com/story/show/168383507)

#### Screenshots (if appropriate)

![homepage](https://user-images.githubusercontent.com/52447234/64540398-6fd2a200-d320-11e9-9e9a-96dc353951bd.PNG)
![signin](https://user-images.githubusercontent.com/52447234/64540447-8a0c8000-d320-11e9-8b33-a6757fa6c3f3.PNG)
